plugin/src/main/java/net/dzikoysk/funnyguilds/data/database/serializer/DatabaseGuildSerializer.java

```diff
@@ -106,7 +111,7 @@ public static Option<Guild> deserialize(ResultSet resultSet) {
             values[8] = Sets.newHashSet();
             values[9] = born;
             values[10] = validity;
-            values[11] = attacked;
+            values[11] = protection;
```

Comment:
Was?? 🍺

Reply:
Currently this field/key/whatever in database are named attacked. There is a plan to change it to`protection` in FG 5.0. So for I renamed variables in code to planned `protection`.